zeeguu_api/api/__init__.py

```diff
@@ -14,4 +14,19 @@
 from . import user_data
 from . import user_settings
 from . import user_statistics
-from . import starred_articles
\ No newline at end of file
+from . import starred_articles
+
+# The code below is for the Flask Dashboard plugin.
+# This code will enable the grouping of unit test results 
+# by the endpoint that they test.
+from flask import request
+import datetime
+import os
+log_dir = os.getenv('DASHBOARD_LOG_DIR')
+@api.after_request
+def after_request(response):
+    t1 = str(datetime.datetime.now())
+    log = open(log_dir + "endpoint_hits.log", "a")
+    log.write("\"{}\",\"{}\"\n".format(t1, request.endpoint))
+    log.close()
+    return response
```

Comment:
you should be able to put this code in the dashboard/__init__.py#bind(app) function... you only seem to need the api variable and that is available at the point where the api calls dashboard.bind(app=app)... so you could simply call dashboard.bind(app=app, blueprints=[api]) and in that way you can monitor even multiple blueprints if needed. 

see what i mean?

Reply:
I see, I will try this out.

Reply:
@kloostert  - Thijs, what's happening with this PR?